src/components/icon-button/icon-button.style.ts

```diff
@@ -13,11 +14,7 @@ const StyledIconButton = styled.button.attrs({ type: "button" })`
     border: none;
     border-radius: var(--borderRadius050);
 
-    &:focus {
-      background-color: transparent;
-      outline: solid 3px var(--colorsSemanticFocus500);
-      z-index: 1;
-    }
+    ${addFocusStyling}
```

Comment:
With the changes I've suggested for the util below, this will need to be:

```
&:focus {
      background-color: transparent;
      ${addFocusStyling}
      z-index: 1;
    }